File: x_f256(score_average)/x_C++/score_average.cpp
#include <cstdio>

int main(void) {
  long long total_score = 0, total_credit = 0;
  int score, credit;
  while (scanf("%d %d", &score, &credit) != EOF) {
    total_score += score * credit;
    total_credit += credit;
  }
  int score_average = total_score / total_credit;
  if (score_average >= 60)
    printf("Oh wow! You pass it!\n%d\n", score_average);
  else printf("YEEEEEE!!!\n");
  return 0;
}

File: x_f256(score_average)/x_Python/score_average.py
from sys import stdin, stdout
total_score = 0
total_credit = 0
while True:
    line = stdin.readline()
    if not line: break
    score, credit = map(int, line.split())
    total_score += score * credit
    total_credit += credit
score_average = total_score // total_credit
if score_average >= 60:
    stdout.write(f'Oh wow! You pass it!\n{score_average}\n')
else: stdout.write('YEEEEEE!!!\n')
